virtual_ecosystem/data_variables.toml

```
@@ -1,31 +1,23 @@
+[[variable]]
+name = "air_heat_conductivity"
+description = "Air heat conductivity between layers"
+unit = "mol m-2 s-1"
+variable_type = "DataArray(float)"
+axis = [space]
```

All of these `space` or `time` throughout this file need to be in quotes, as they are strings:
```suggestion
axis = ["space"]
```